public/index.html

```diff
@@ -2,13 +2,21 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
```

Comment:
I'm not actually sure how to confirm that this is working until we're able to set up and deploy to a new Netlify account... does anyone know if there's a way to test it at this time?